common/djangoapps/student/views/dashboard.py

```diff
@@ -759,3 +770,22 @@ def student_dashboard(request):
     response = render_to_response('dashboard.html', context)
     set_user_info_cookie(response, request)
     return response
+
+
+def _sort_preferred_order_by_course_enrollments(order_by, course_enrollments):
```

Comment:
I don't want to nit-pick, but "sort preferred order by course enrollments" doesn't make sense.  We aren't sorting preferred orders, and we certainly aren't sorting them by course enrollments.  This function has nothing to do with preferences, it sorts course enrollments in the way that you ask it to.  Is "_sort_course_enrollments" a bad name?

Reply:
Bit-pick all you want, your suggestion makes more sense than how I named the function. Gonna update the name.